conbench/tests/populate_local_conbench.py

```diff
@@ -392,9 +398,13 @@ def create_benchmarks_with_history():
                         mean=mean,
                     )
 
-                post_benchmarks(benchmark_data)
+                benchmark_id = post_benchmarks(benchmark_data)
+                benchmark_ids.append(benchmark_id)
                 runs.append((run_id, timestamp))
 
+    log.info("now, emit a benchmark ID on stdout")
+    print(benchmark_ids[0])
```

Comment:
Would you mind commenting on why we are sending the id to stdout instead of log here?

Reply:
In this PR here it's used in CI via `export SOME_BENCHMARK_ID="$(make -s db-populate)"`. 

Could have added a comment of course, but I don't expect this particular code here to stay around for too long though -- https://github.com/conbench/conbench/pull/611/files#r1082918585

Reply:
added a code comment

Reply:
> In this PR here it's used in CI via export SOME_BENCHMARK_ID="$(make -s db-populate)".

AAAAH thanks for this. I grepped for the script name, but hadn't looked in the Makefile. That makes sense (and also very much explains the split behavior, now!)